### ISSUE TITLE
Displaying displaying files keeps showing the same file. 

### ISSUE THREAD

**Issue by macdevnet** [ZeeZide/Shrugs#17]
When you Quicklook a file that has been posted in a channel after that Quicklook will continue to display that file even when you are trying to view a different one.  This can continue even after a restart.  It does sometimes stop and then display the correct file but I cannot work out what common thing is needed to make it do that. 

Screen recording attached. 

[ScreenRecording.zip](https://github.com/ZeeZide/Shrugs/files/4937196/ScreenRecording.zip)

**Comment by helje5** [comment]
Hm, yes, the QuickLook support sometimes seem to have some issue w/ not picking the right one.
(careful w/ posting such documents! :-) )

In this one I think what's happening is that Shrugs is opening the right files, but both of them (with the first one coming up). Can you navigate to the second using arrows keys (or the <> buttons in the quicklook)?
Even if this works, this still needs to be fixed, so that the clicked item is preselected.

**Comment by macdevnet** [comment]
The documents are from a charity and will be included in the meeting minutes that will be public anyway so there is no real risk in posting them here.

**Comment by macdevnet** [comment]
Yes, it does seem to have opened both documents (or at least left the first one open)  and you can access the second via the arrow keys.
